doc/api/stream.md

```diff
@@ -471,6 +471,15 @@ process.nextTick(() => {
 
 See also: [`writable.cork()`][].
 
+##### writable.destroyed
+<!-- YAML
+added: REPLACEME
```

Comment:
"REPLACEME" should be the Node.js version where this was actually introduced, not the coming one.

Reply:
```suggestion
added: v8.0.0
```

It was added in PR https://github.com/nodejs/node/pull/12925